package/com.unity.formats.usd/Tests/Editor/EditorTests.cs

```diff
@@ -9,5 +12,35 @@ public void DummyTest()
         {
         }
     }
+    
+    public class ScopeLoadingTest
+    {
+        private GameObject usdRoot;
+        private string filepath = "test_#154/test_collections.usda";
+        
+        [SetUp]
+        public void SetUp()
+        {
+            InitUsd.Initialize();
+            var stage = pxr.UsdStage.Open(Path.Combine(Application.dataPath, filepath), pxr.UsdStage.InitialLoadSet.LoadNone);
+            var scene = Scene.Open(stage);
+            usdRoot = USD.UsdMenu.ImportSceneAsGameObject(scene);
```

Comment:
Should probably make a SceneLoader utility class (either in this change or a future change when adding new tests), so all scripts don't have to implement this logic

Reply:
That's what I've been telling myself for each branch I've worked on so far... 
I need to rewrite the loading part anyway because assets are not found on Yamato. Adding them to the package and accessing them through guid seems to be the correct way.

Reply:
+1. 

You can make a base framework to do that for you. That will probably become very redundant. Plus, you delegate the knowledge of "How do I load my stuff on Yamato" to one place, allowing people to focus on their tests rather than Yamato-specific rules.

Reply:
I'll create a separate task for that